week-1-practise/level-3/1-piping.js

```diff
@@ -11,6 +11,7 @@
 */
 
 var startingValue = 5;
+var result = "£" + (startingValue * 2 + 10).toFixed(2);
```

Comment:
Check the instructions above. The aim is to use the supplied functions rather than writing the code yourself.